### PR TITLE
Block remove chunk update priority

### PR DESCRIPTION
**Pull request by adam-kucz**

### CODE REVIEW

File: Scripts/Util/HashDeque.cs [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    public T GetFirst()
+    {
+        return deque.First.Value;
+    }
+
+    public T RemoveLast()
+    {
+        T item = deque.Last.Value;
+        deque.RemoveFirst();
```
Comment (JamesHinshelwood) [inline Scripts/Util/HashDeque.cs]: uuuhhh....

Comment (JamesHinshelwood) [reply Scripts/Util/HashDeque.cs]: RemoveLast?

**Review by JamesHinshelwood — APPROVED** [review]
Other than this, everything is fine, so approving, but pls fix

File: Scripts/Util/HashDeque.cs (hunk 1 above)
Comment (adam-kucz) [reply Scripts/Util/HashDeque.cs]: it's not used anywhere anyway

**Pull request merged**